src/pages/ReimbursementAccount/IdentityForm.js

```diff
@@ -171,14 +171,14 @@ function IdentityForm(props) {
                     />
                 </View>
             </View>
-            <DatePicker
+            <InputWrapper
+                InputComponent={DatePicker}
                 inputID={props.inputKeys.dob}
                 shouldSaveDraft={props.shouldSaveDraft}
                 label={`${props.translate('common.dob')}`}
                 containerStyles={[styles.mt4]}
                 placeholder={props.translate('common.dateFormat')}
                 defaultValue={props.values.dob || props.defaultValues.dob}
-                onInputChange={(value) => props.onFieldChange({dob: value})}
```

Comment:
I checked and both `props.onFieldChange` and `props.values` from `IdentityForm` aren't used anywhere. So I would get rid of them completely. WDYT?

Reply:
That's sounds good. Let's remove `onFieldChange` from `AddressForm` as well. But we may keep `values` for now (just feel like it should be there since defaultValues is there - no strong preference anyway)